src/main/java/edu/greenblitz/robotName/commands/shooter/StopFunnel.java

```diff
@@ -0,0 +1,8 @@
+package edu.greenblitz.robotName.commands.shooter;
+
+public class StopFunnel extends FunnelCommand {
```

Comment:
there is no reason this should be a command. unlike the flywheel you don't need to have a command to stop this, because the funnel does not run all of the time.